### PR TITLE
WebApi controller

### PR DESCRIPTION
**Pull request by Tdallau**
I created a command for generating a dotnet core webapi controller.

### CODE REVIEW

**Comment by PoLaKoSz** [comment]
What do You think about decorating the class with the [`[ApiController]`](https://docs.microsoft.com/en-us/aspnet/core/web-api#apicontroller-attribute) attribute?

**Comment by Tdallau** [comment]
good point, i just added it.

File: package.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
     "name": "csharpextensions",
     "displayName": "C# Extensions",
     "description": "C# IDE Extensions for VSCode",
-    "author": "KreativJos",
```
Comment (KreativJos) [inline package.json:5]: Would you please not change the author & publisher names, and the repository in a pull-request?

File: publish.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ if [[ $test_output == *"failing"* ]]; then
     echo "$test_output"
 else
     vsce package
-    sed -e 's/"publisher": "kreativ-software"/"publisher": "jsw"/g' ./package.json >./package.json.temp
+    sed -e 's/"publisher": "tdallau-csharpextensions"/"publisher": "jsw"/g' ./package.json >./package.json.temp
```
Comment (KreativJos) [inline publish.sh:10]: Why did you change the publisher here? I don't see any packages in the vscode extensions that you authored:
image

File: src/extension.ts [hunk 3]
```diff
@@ -75,7 +80,7 @@ function correctExtension(filename: string) {
 
 function openTemplateAndSaveNewFile(type: string, namespace: string, filename: string, originalfilepath: string) {
     const templatefileName = type + '.tmpl';
-    const extension = vscode.extensions.getExtension('kreativ-software.csharpextensions');
+    const extension = vscode.extensions.getExtension('tdallau-csharpextensions.csharpextensions');
```
Comment (KreativJos) [inline src/extension.ts:83]: Same as above, should not be in this pull-request

**Review by KreativJos — CHANGES_REQUESTED** [review]
The main issues are about the repository name, publisher name etc which should not be in a pull request to this repository.

File: publish.sh (hunk 2 above)
Comment (pm4rcin) [reply publish.sh:10]: He made [that extension](https://marketplace.visualstudio.com/items?itemName=tdallau-csharpextensions.csharpextensions). Check the link url his nickname is there.

Comment (KreativJos) [reply publish.sh:10]: Thnx for pointing out my error! 😊 I have been to flimsy in my research it seems.

**Comment by pm4rcin** [comment]
@Tdallau would you like to rebase the PR and make needed changes?

**Comment by KreativJos** [comment]
The requested change has been implemented in https://github.com/KreativJos/csharpextensions/pull/30
Closing this pull-request.
